ddp-workspace/projects/ddp-dsm-ui/src/app/WELCOME/auth/auth.component.ts

```diff
@@ -43,12 +44,24 @@ export class AuthComponent implements OnInit {
   popUpIsShown = true;
   authError: Observable<string | null>;
   pickList$: Observable<any>;
+  private source$: Observable<StorageEvent>;
 
   constructor(private auth: Auth, private title: Title, private router: Router) {}
 
   ngOnInit(): void {
+
+    this.source$ = fromEvent<StorageEvent>(window, 'storage');
+    this.source$.subscribe(
```

Comment:
I would suggest (It's the best and recommended practice) unsubscribing any subscribed observable (e.g. `this.source$`) in order to avoid memory leaks and unexpected behavior in the future.